packages/p2p/src/components/my-profile/payment-methods/payment-methods-list/edit-payment-method-form.jsx

```diff
@@ -1,16 +1,27 @@
 import React from 'react';
 import { observer } from 'mobx-react-lite';
-import { Field, Form, Formik } from 'formik';
+import { Field, Form, Formik, useFormikContext } from 'formik';
 import { Button, DesktopWrapper, Input, Loading, Modal, Text } from '@deriv/components';
 import { Localize, localize } from 'Components/i18next';
 import { usePaymentMethodValidator } from 'Components/hooks';
 import { useStores } from 'Stores';
 import CancelEditPaymentMethodModal from './cancel-edit-payment-method-modal.jsx';
 import PageReturn from 'Components/page-return/page-return.jsx';
-import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
-const EditPaymentMethodForm = ({ formik_ref }) => {
+const RestoreFormHistory = () => {
```

Comment:
Since `RestoreFormHistory` Component is also being used in add payment method form, could we separate the component out?